.claude/settings.local.json

```diff
@@ -20,8 +20,7 @@
       "mcp__serena__find_file",
       "mcp__serena__write_memory",
       "Bash(fvm dart:*)",
-      "mcp__dart__run_tests",
-      "Bash(dart run:*)"
```

Comment:
Revert this bro. 

      "mcp__dart__run_tests",
      "Bash(dart run:*)"
      
      Need to allow both